.laminas-ci.json

```diff
@@ -4,7 +4,6 @@
         "--no-plugins"
     ],
     "ignore_php_platform_requirements": {
-        "8.0": false,
-        "8.1": false
+        "8.2": true
```

Comment:
We shouldn't have this on the skeleton: this means trouble for people installing the skeleton :thinking:

Reply:
Hmm, if 8.2 does not apply cleanly, then perhaps yes.

My question here was rather whether CI would work without this specific `true`, and whether we can instead do a couple more releases to get there instead.

Reply:
Ok, so indeed it seems like mezzio-tooling is the last missing piece

Reply:
Bumped mezzio/mezzio-tooling to ^2.8